File: examples/standalone/pipelineYaml/snippets/python/iot-augment-alerts.py
import json
obj = json.loads(input)
if obj.get("averageTemperature", 0.0) > 0.0:
    obj["STATUS"] = "CRITICAL_ALERT"
    obj["message"] = "Refrigeration failure detected! Cargo at risk."
else:
    obj["STATUS"] = "NORMAL"
return json.dumps(obj)

File: examples/standalone/pipelineYaml/snippets/java/iot-augment-alerts.java
com.fasterxml.jackson.databind.ObjectMapper mapper = new com.fasterxml.jackson.databind.ObjectMapper();
com.fasterxml.jackson.databind.node.ObjectNode obj = (com.fasterxml.jackson.databind.node.ObjectNode) mapper.readTree(input);
if (obj.get("averageTemperature").asDouble() > 0.0) {
    obj.put("STATUS", "CRITICAL_ALERT");
    obj.put("message", "Refrigeration failure detected! Cargo at risk.");
} else {
    obj.put("STATUS", "NORMAL");
}
return obj.toString();
